src/modules/fundingManager/oracle/interfaces/IFM_PC_ExternalPrice_Redeeming_v1.sol

```diff
@@ -0,0 +1,185 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+pragma solidity ^0.8.0;
+
+// Internal
+import {IFundingManager_v1} from "@fm/IFundingManager_v1.sol";
+import {IERC20PaymentClientBase_v1} from
+    "@lm/interfaces/IERC20PaymentClientBase_v1.sol";
+import {IRedeemingBondingCurveBase_v1} from
+    "@fm/bondingCurve/interfaces/IRedeemingBondingCurveBase_v1.sol";
+
+/**
+ * @title   External Price Funding Manager Interface
+ *
+ * @notice  Interface for a funding manager that uses external price feeds for token
```

Comment:
Needs to be the same as in the contract, this one is old.

Reply:
weird, I remember changing this. Done